server/grpc.go

```diff
@@ -76,7 +76,16 @@ func (g *GRPCServer) Start() error {
 		return err
 	}
 
-	return g.grpcServer.Serve(lis)
+	g.logger.
+		WithField("port", g.port).
+		Infof("✅  Started gRPC server on port %d", g.port)
+
+	err = g.grpcServer.Serve(lis)
+	if err != nil {
+		return err
+	}
+
```

Comment:
should the log be after the err handling

Reply:
Serve is a blocking call. Basically if we opened the port for listening, kernel should queue incoming connections and requests, So if we can listen the port, we assume started.

Reply:
oh right sorry, forgot.